docs/deployment/getting-started.md

```diff
@@ -0,0 +1,76 @@
+# Deployment
+
+Server deployment is a complex area, that will depend on what kind of service you're deploying Uvicorn onto.
+
+### TL;DR
+As a general rule, you probably want to:
+
+* Run `uvicorn --reload` from the command line for local development.
+* Run `gunicorn -k uvicorn.workers.UvicornWorker` for production (non containerized).
+* Run behind Nginx for self-hosted deployments.
```

Comment:
why for self-hosted deplyments, the use of nginx could be totally justified in a good number of other cases, this doesnt seem right to me